### ISSUE TITLE
Not working with Node 8

### ISSUE THREAD

**Issue by davelosert** [virtualidentityag/gulp-build-framework#23]
When trying to `yarn start` with Node Version 8.0.0, I get this error:

```
/home/davidlosert/Projects/ifx-infineon/ifx-infineon-com-cms-frontend/node_modules/require-dir/index.js:97
            if (!require.extensions.hasOwnProperty(ext)) {
                                    ^

TypeError: require.extensions.hasOwnProperty is not a function
    at requireDir (/home/davidlosert/Projects/ifx-infineon/ifx-infineon-com-cms-frontend/node_modules/require-dir/index.js:97:37)
    at Object.<anonymous> (/home/davidlosert/Projects/ifx-infineon/ifx-infineon-com-cms-frontend/gulpfile.js:5:1)
```

This is probably fixed easiest by updating the dependency.
I will try to set up a PR for this today.

**Comment by davelosert** [comment]
Okay, unfortunately, gulp-git-push has also a sub-dependency on a really old version of require-dir (0.1.0) which still causes the bug. That doesn't seem to be fixable so easy unfortunately.

It makes me wonder: Isn't npm / yarn supposed to always take just one of the packages?
Anyways: I wonder if it is a good idea to have depenbdencies in this framework on a package with only 2 Stars and updates longer than 1 year ago...

**Comment by timomayer** [comment]
@alxbenz we can remove the "release" task i wrote since it is not used very often so this dependency will be gone and the build framework should run with node 8

**Comment by janrembold** [comment]
This is already fixed in 5.0 branch. So closing it.
